samples/server/go-server/api/server/api/api.go

```diff
@@ -97,8 +99,16 @@ func Run(config *Config, storage *server.Storager) {
 		log.Println("grpc server is configured without client certificate authentication")
 	}
 
+	if len(config.CORSAllowedOrigins) > 0 {
```

Comment:
I would prefer CORS to be disabled by default. It's easy enough to enable in the config.

Reply:
Done.